addons/website_sale_loyalty/static/src/js/portal_loyalty_card.xml

```diff
@@ -16,7 +16,7 @@
                     <span class="text-primary" t-out="reward.points"/>
                 </div>
             </div>
-            <div class="d-flex justify-content-center">
+            <div class="d-flex justify-content-center" t-if="props.rewards.length > 0">
```

Comment:
Nitpicking, but `t-if` attributes should be the first on a node as it's the most important attribute because it decides whether the content is there or not.